test/ParserTests.hs

```diff
@@ -627,6 +630,25 @@ testTypeExtLimitation2 = TestCase (
   False
   (isRight $ parseAll (parseGame []) "" "game E\ntype T1 = Board\ntype T2 = Int & T1"))
 
+testCantDefineContentBefore :: Test
+testCantDefineContentBefore = TestCase (
+  assertBool "Test that Content cannot cannot defined by a user before board"
```

Comment:
You should fix these test messages before we take this in, all 3 are the same.

Reply:
:facepalm: Never copy and paste code, folks!

Reply:
In my defense, all three were different. But I propagated the "cannot cannot" typo.

Reply:
Yeah I think I'm tired...I just read it multiple times and somehow still missed it until I saw your diffs, minus the double word. But they're very clearly different 😆 .